io.github.brainstormtrooper.facteur.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id" : "io.github.brainstormtrooper.facteur",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.brainstormtrooper.facteur",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=ssh-auth",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=home:create",
+        "--filesystem=xdg-run/gvfsd",
```

Comment:
if you really need gvfs access, this is insufficient.

if you don't need gvfs, then thise should be removed.

https://docs.flatpak.org/en/latest/sandbox-permissions.html#gvfs-access

Reply:
removed